solr/bin/solr.cmd

```diff
@@ -94,7 +94,10 @@ IF NOT DEFINED SOLR_SSL_RELOAD_ENABLED (
   set "SOLR_SSL_RELOAD_ENABLED=true"
 )
 
-REM Enable java security manager by default (limiting filesystem access and other things)
+REM Enable java security manager by default for Java 23 and before (limiting filesystem access and other things)
+IF !JAVA_MAJOR_VERSION! GEQ "24" (
```

Comment:
I've tested again the Windows script changes, seems like the condition is always true, regardless the java version.

This should fix the issue:

```suggestion
IF !JAVA_MAJOR_VERSION! GEQ 24 (
```

Note that it matter whether you set `JAVA_HOME` or update the `PATH` variable.